kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -29,7 +29,8 @@ import (
 	"k8s.io/kubeadm/kinder/pkg/constants"
 	"k8s.io/kubeadm/kinder/pkg/cri"
 	"k8s.io/kubeadm/kinder/pkg/kubeadm"
-	kindkustomize "sigs.k8s.io/kind/pkg/kustomize"
+	kindkustomize "k8s.io/kubeadm/kinder/third_party/kind/util/kustomize"
+	"sigs.k8s.io/kind/pkg/apis/config/v1alpha3"
```

Comment:
[1] We are always adding aliases for kind imports. This can be 
`kindV1alpha3 "sigs.k8s.io/kind/pkg/apis/config/v1alpha3"`

Reply:
my idea with this file was to not import kind versions directly and it as a wrapper:
https://github.com/kubernetes/kubeadm/blob/master/kinder/pkg/config/config.go

currently it's the only place where kind's v1alpha3 is imported, which makes changes in kinder public types easier to manage in kinder.

Reply:
aliasing v1alpha3.PatchJSON6902 in there should be sufficient.

Reply:
isn't the only nesting here: https://github.com/kubernetes-sigs/kind/blob/master/pkg/apis/config/v1alpha3/types.go#L51
?